### PR TITLE
Card

### PR DESCRIPTION
**Pull request by anandkumarcoder**
Feat:Card
Link: https://component-library-mlq8ugl1e-anandkumarcoder.vercel.app/docs/docs.html
image
image
image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (nikhil-sharma-nks) [inline .vscode/settings.json:1]: if this file is not necessary then you should consider it moving in gitignore file

File: docs/docs.html [hunk 2]
```diff
@@ -365,25 +361,279 @@ <h4>Example of icon buttons</h4>
...
+      <div class="card-section" id="card">
+        <div class="heading p-xs mb-sm">
+          <h2>Cards</h2>
+          <h5>cards are important part od many ecommerce websites</h5>
+        </div>
+
+        <div class="description-box mb-sm">
+          <h3>Example</h3>
+          <div class="text-description p-xs mb-sm">
+            <p>Simple card/p></p>
```
Comment (nikhil-sharma-nks) [inline docs/docs.html:650]: remove extra closing p tag it is coming as text in the layout

**Review by nikhil-sharma-nks — COMMENTED** [review]
consider making other kind of cards which are mentioned in the Component Library PRD.
Other than that looks good to me. Nice work 👍

**Review by darshandamre — COMMENTED** [review]
take care of the things nikhil mentioned, rest looks good

**Pull request merged**